### ISSUE TITLE
Angular ui-select not displaying

### ISSUE THREAD

**Issue by lvasilev1** [angular-ui/ui-select#1902]
I want to use ui-select, but it is not displaying anything and there are no errors in the developer console. I've been googling this for a few hours now and I've tried all the suggestions I've found, but nothing has helped.

I use Chrome and I've included Bootstrap version 3.3.4 and Angular version 1.4.8 in my SPA project. I've also included the latest versions of select.min.css and select.min.js. I added ui-select (version 0.19.5) to my project through NuGet in Visual Studio 2015.

I've reduced my code to a very simple example while trying to isolate the problem. Here is what I have right now (this is in my view):

  `' <div` id="damageType" name="damageType" `class="col-md-4">`
    `    <ui-select` ng-maxlength="10">
            `<ui-select-choices` repeat="f in ['light','hpexp']">
                `{{` f }}
            `</ui-select-choices>`
        `</ui-select>`
  `</div>'`

I've tried adding a ng-hide, like so, to no avail:

  `' <div` id="damageType" name="damageType" `class="col-md-4">`
    `    <ui-select` ng-maxlength="10">
            `<ui-select-choices` repeat="f in ['light','hpexp']" ng-hide="!$select.open">
                `{{` f }}
            `</ui-select-choices>`
        `</ui-select>`
  `</div>'`

Here is what it looks like in the page. The div where the drop-down should be displayed is there, but it's height is 1. Also, as you can see, there are no errors in the console:

![ui_select_issue](https://cloud.githubusercontent.com/assets/16479217/22483059/49752530-e7c1-11e6-8731-476848118906.png)

What could be the problem here?

**Comment by ckosloski** [comment]
I don't see a `ng-model` on your `ui-select`.
Maybe instead of `{{f}}` try `<span ng-bind="f"></span>`

**Comment by Jefiozie** [comment]
Hi, @lvasilev1  Can you make a example plunker for this? People can then have a better look at your issue. For now I don't see any problems , did you forgot to include the lib?

**Comment by lvasilev1** [comment]
@ckosloski @Jefiozie Yea, I didn't have ng-model on it, cause I thought that was not necessary if you just wanted to display some values and not tie them to an object in the controller.

So I added that and also in my mainApp I didn't have dependencies on "ui.select" and "ngSanitize". Once I did all that and also included the angular-sanitize.min.js in my index Html file, things started working.

Thanks!
